assets/svg/zodiac/aquarius.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="0 0 24 30" xml:space="preserve">
```

Comment:
`xmlns:xlink="http://www.w3.org/1999/xlink"` this svg extension is not used in this file we can delete it